config.xhgui.yaml

```diff
@@ -0,0 +1,21 @@
+#ddev-generated
+hooks:
+  post-start:
+  - exec: |
+      cat <<-PHP > .ddev/xhprof/xhprof_prepend.php
```

Comment:
Wouldn't it be easier to use a `cp` here instead of cat? Then it's also one line.